app/components/circularDisplay/FrontMatter.tsx

```diff
@@ -68,7 +68,11 @@ export function FrontMatter({
           </small>
         </FrontMatterItem>
       )}
-      <FrontMatterItem label="From">{submitter}</FrontMatterItem>
+      <FrontMatterItem label="From">
+        <Link href={`/circulars?query=submitter:"${submitter}"`}>
```

Comment:
- Is there a function that you can call to correctly escape the submitter for Lucene?
- Please use the [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL_API) and [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) to construct the URL to ensure that query string arguments are correctly encoded.